caa2021_packages_abstract.md

```diff
@@ -1,33 +1,38 @@
 ---
-title: "Tools for the revolution: developing scientific programming packages for archaeology"
+title: "Tools for the 'Third Science Revolution': developing scientific software packages for archaeology"
```

Comment:
I like the 'software', I do not know if the Kristiansen reference is to not to bold...

Reply:
Well - I think it's not too far-fetched. Kristiansen is explicitly writing about how he expects computational, quantitative analysis  to change the field and we are the coding infantry that powers some of this transformation. This title attracts attention, I guess.

Reply:
OK.

Reply:
Let's be bold. What's the worst that could happen...

But I'd keep "scientific programming" because I'm worried that otherwise the scope (R, Python & friends) is not clear. "Scientific software" could include e.g. pure point-and-click software, mobile apps...